libs/store_helper/user/ebpf_registry_helper.cpp

```diff
@@ -18,6 +19,25 @@
 ebpf_store_key_t ebpf_store_root_key = HKEY_CURRENT_USER; // TODO: Issue #1231 Change to using HKEY_LOCAL_MACHINE
 const wchar_t* ebpf_store_root_sub_key = EBPF_ROOT_RELATIVE_PATH;
 
+wchar_t*
+ebpf_get_wstring_from_string(_In_ const char* text)
+{
+    int length = MultiByteToWideChar(CP_UTF8, 0, text, -1, nullptr, 0);
+    wchar_t* wide = (wchar_t*)ebpf_allocate(length * sizeof(wchar_t));
+    if (wide == nullptr) {
+        return nullptr;
+    }
+    MultiByteToWideChar(CP_UTF8, 0, text, -1, wide, length);
```

Comment:
Can this be a wrapper around _get_wstring_from_string(std::string text)?  e.g., something like:
```
std::wstring wide = _get_wstring_from_string(text);
return ebpf_duplicate_wstring(wide.c_str());
```

Reply:
Actually `_get_wstring_from_string()` is no longer needed. Deleted `_get_wstring_from_string()`